src/IntrType.f90

```diff
@@ -98,6 +98,7 @@ MODULE IntrType
   PUBLIC :: SOFTEQ
   PUBLIC :: SOFTEQR
   PUBLIC :: SOFTLE
+  PUBLIC :: SOFTLT
```

Comment:
Implemented SOFTGT, but there doesn't appear to be a direct test for IntrTypes. Might punt that to a separate issue.